packages/cli/src/commands/init/templateName.js

```diff
@@ -81,7 +81,13 @@ async function tryTemplateShorthand(templateName: string) {
       `https://registry.yarnpkg.com/${reactNativeTemplatePackage}/latest`,
     );
 
-    if (JSON.parse(response).name) {
+    if (response.status < 200 || response.status > 299) {
+      throw new Error(`Failed to load page, status code: ${response.status}`);
+    }
+
+    const body = await response.json();
```

Comment:
can't we do the `response.json` inside fetch function?

Reply:
Yes, but sometimes we check for status of the request. Maybe we should add generic handler if the response status code is not 2xx?

Reply:
we can check for status inside fetch and then return something like `{data: response.json(), status}`

Reply:
That makes sense. On it.